CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 ### Features
 
-- None
+- Add ValidationFailed state and signal in anticipation of validating task outputs [#2143](https://github.com/PrefectHQ/prefect/issues/2143)
```

Comment:
should I have put this in enhancements? 🤔

Reply:
Yea I classify this as an enhancement; I typically think of "Features" as headline things worth socializing

Reply:
Gotcha, moved it in 39629e5